.github/CODEOWNERS

```diff
@@ -81,21 +81,22 @@
 /moveit_ros/move_group/	                                    @rhaschke @IanTheEngineer
 /moveit_ros/visualization/	                            @rhaschke @jonbinney @christian-rauch
 
-/moveit_ros/planning/collision_plugin_loader/               @rhaschke
-/moveit_ros/planning/constraint_sampler_manager_loader/     @henningkayser
+/moveit_ros/planning/collision_plugin_loader/               @j-petit
+/moveit_ros/planning/constraint_sampler_manager_loader/     @nbbrooks
 /moveit_ros/planning/kinematics_plugin_loader/              @gavanderhoorn
 /moveit_ros/planning/plan_execution/                        @v4hn
 /moveit_ros/planning/planning_components_tools/             @henningkayser
 /moveit_ros/planning/planning_pipeline/                     @v4hn
 /moveit_ros/planning/planning_request_adapter_plugins/      @v4hn
 /moveit_ros/planning/planning_scene_monitor/                @rhaschke
-/moveit_ros/planning/rdf_loader/                            @henningkayser
-/moveit_ros/planning/robot_model_loader/                    @henningkayser
+/moveit_ros/planning/rdf_loader/                            @nbbrooks
+/moveit_ros/planning/robot_model_loader/                    @nbbrooks
 /moveit_ros/planning/trajectory_execution_manager/          @rhaschke
 
-/moveit_setup_assistant/	                            @davetcoleman @rhaschke @MohmadAyman
+/moveit_setup_assistant/	                            @RoboticsYY @rhaschke @MohmadAyman
 
 /moveit_planners/ompl/	                                    @BryceStevenWilley @mamoll
 /moveit_planners/chomp/chomp_interface/                     @raghavendersahdev @knorth55 @bmagyar
 /moveit_planners/chomp/chomp_optimizer_adapter/	            @raghavendersahdev @knorth55 @bmagyar
 /moveit_planners/chomp/chomp_motion_planner/	            @raghavendersahdev @knorth55 @bmagyar
+/moveit_planners/trajopt                                    @ommmid @mamoll
```

Comment:
@ommmid added trajopt to moveit, so is the new reviewer, along with motion planning expert @mamoll